src/components/Alert.js

```diff
@@ -1,26 +1,41 @@
 var React = require('react');
 var classNames = require('classnames');
+var utils = require('../utils');
 
 var ALERT_TYPES = [
 	'danger',
 	'error', // alias for danger
 	'info',
-	'primary',
 	'success',
-	'warning',
+	'warning'
 ];
 
 module.exports = React.createClass({
 	displayName: 'ElementalAlert',
 	propTypes: {
 		children: React.PropTypes.node.isRequired,
 		className: React.PropTypes.string,
-		type: React.PropTypes.oneOf(ALERT_TYPES).isRequired,
+		theme: utils.getThemeValidator({
+			classes: ALERT_TYPES
+		}),
+		type: React.PropTypes.oneOf(ALERT_TYPES).isRequired
+	},
+	getDefaultProps () {
+		return {
+			theme: {
+				classes: {
+					danger: 'Alert Alert--danger',
+					error: 'Alert Alert--danger',
```

Comment:
both works, can change it to error, in general its a bad thing to have different names for the same thing.